### ISSUE TITLE
Comment text is difficult to read in Firefox Dark Mode

### ISSUE THREAD

**Issue by anoadragon453** [krixano/ZeroExchange#3]
![image](https://user-images.githubusercontent.com/1342360/55096708-4b587580-50b2-11e9-862c-692ae8074cf8.png)

Firefox 60.6.0, using built-in dark theme.

I've noticed this happen with a few sites, and it doesn't happen in Chrome, but it's fixable by just setting .textarea's CSS to `color: black;`.

**Comment by krixano** [comment]
Yeah, when I'm on linux distros that use dark themes I get this type of problem literally *all the time* on tons of different websites. I'll get this fixed when I start working on ZeroExchange again. Thanks!
